integration-tests/integration/externaluser.search.spec.js

```diff
@@ -269,6 +286,7 @@ context('External user search functionality', () => {
       const validateCsv = (list) => {
         expect(list, 'number of records').to.have.length(22)
         expect(list[0], 'header row').to.deep.equal([
+          'userId',
```

Comment:
I checked that these were indeed included in the CSV in dev, just missing in the tests but shows up correctly now.